docs/modules/ROOT/pages/reference/jetty.adoc

```diff
@@ -23,4 +26,16 @@ resourceConfig.register(new MetricsApplicationEventListener(
                 "http.server.requests",
                 true));
 ServletContainer servletContainer = new ServletContainer(resourceConfig);
-----
\ No newline at end of file
+----
+
+[[overview-observation]]
+== Eclipse Observation Jersey Instrumentation
+
+Below you can find an example of how to instrument Jersey with https://docs.micrometer.io/micrometer/reference/observation.html[Micrometer Observation]. That means that depending on your Observation Handler configuration you instrument once, and can have multiple benefits out of it (e.g. metrics, distributed tracing).
```

Comment:
Should we not use relative links to the Observation docs rather than the absolute link?